haskell/nettest/StablecoinClientTest.hs

```diff
@@ -34,23 +33,21 @@ stablecoinClientScenario = do
   (contractOwnerAlias, contractOwnerAddr, contractOwner) <- createRole "contract-owner"
   (_, _, minter) <- createRole "minter"
   (transferlistAlias, transferlistAddr, transferlist) <- createRole "transferlist"
-  (_, _, mdRegisty) <- createRole "metadata-registry"
+  (_, _, _) <- createRole "metadata-registry"
```

Comment:
Why not remove the whole `createRole` call?

Reply:
Removed